src/dsl.ts

```diff
@@ -86,7 +88,7 @@ export function parse(input: string): ParseResult {
       ? {
           kind: "schema",
           stringValue: input,
-          definitions: (result as Success<Array<TopLevelDefinition>>).value,
```

Comment:
Fixing the types elsewhere let me get rid of this, so we get better type safety.